var/spack/repos/builtin/packages/libcatalyst/package.py

```diff
@@ -14,10 +18,16 @@ class Libcatalyst(CMakePackage):
     git = "https://gitlab.kitware.com/paraview/catalyst.git"
     url = "https://gitlab.kitware.com/api/v4/projects/paraview%2Fcatalyst/packages/generic/catalyst/v2.0.0/catalyst-v2.0.0.tar.gz"
 
-    maintainers("mathstuf")
-
+    maintainers("mathstuf", "ayenpure")
+    version("master", branch="master")
+    version(
+        "2.0.0-rc4",
+        sha256="cb491e4ccd344156cc2494f65b9f38885598c16d12e1016c36e2ee0bc3640863",
+        preferred=True,
+    )
     version("2.0.0-rc3", sha256="8862bd0a4d0be2176b4272f9affda1ea4e5092087acbb99a2fe2621c33834e05")
-
+    version("2.0.0-rc2", sha256="661ae7c630ff6e382dba45b90bc27e7440e0ed27a2c9dec23c053cec4c7d9507")
+    version("2.0.0-rc1", sha256="451952ca196b904347e9e03a66dedc335d4420d71f3e80d6695aea22fa12e824")
```

Comment:
Why are we adding older release candidates? Aren't these versions supposed to be "ephemeral" and disappear once v2.0.0 is released?

Reply:
I think you are right, we should probably only have the latest available here.

On that note, the datestamp release I don't think is usable anymore, it is more of a Beta/Alpha release, so we could drop that as well.

Reply:
I've removed all the previous versions.